src/engine/renderer/tr_bsp.cpp

```diff
@@ -484,7 +484,7 @@ static void R_LoadLightmaps( lump_t *l, const char *bspName )
 			}
 			if ( hdrFiles.empty() )
 			{
-				Log::Warn("no lightmap files found");
+				Log::Warn("no lightmap file found");
```

Comment:
The original is more correct.

Reply:
Why? Is “none” plural in English?

Reply:
`no lightmap file found` suggests that you were only expecting to find one.

Reply:
Wow, I would have expected that meaning from `lightmap file not found` instead.

Reply:
Yes that also would suggest seeking only one

Reply:
I removed that commit.